nncf/quantization/algorithms/smooth_quant/torch_backend.py

```diff
@@ -161,6 +175,10 @@ def get_weight_channel_axis(node: NNCFNode) -> int:
 
     @staticmethod
     def is_node_with_shared_weight(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
+        if is_experimental_torch_tracing_enabled():
+            weight_node = get_const_node(node, node.metatype.weight_port_ids[0], nncf_graph)
+            output_edges = nncf_graph.get_next_nodes(weight_node)
+            return len(output_edges) > 1
         return node.is_shared()
```

Comment:
My opinion, `node.is_shared()` is simpler to use instead of code which calculates the number of output edges. Are you going to add this information during creating `NNCFGraph`?

Reply:
no, detection shared weight align with ov
is_shared attribute is currently not set or used.